src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+    private final String color;
```

Comment:
you should move this field to Figure and initialize it using Figure's constructor inside of concrete figure's constructor, the same for other figures